### DISCUSSION TITLE
Transfer  cake-dotnet-vulnerability-scanner and cake-cloudformation to cake-contrib

### DISCUSSION THREAD

**Discussion by zahycs** [cake-build/cake#4114] [Extension requests]
Hi @augustoproiete 

Thanks for reaching out through email, We would like to transfer these 2 repos to cake_contrib 

1. https://github.com/epbloxx/cake-cloudformation
2. https://github.com/epbloxx/cake-dotnet-vulnerability-scanner

**Comment by augustoproiete** [comment]
Thanks @zahycs I sent you instructions via email.
